ts/sagas/services/handleServiceReadabilitySaga.ts

```diff
@@ -0,0 +1,20 @@
+import { SagaIterator } from "redux-saga";
+import { put, select } from "redux-saga/effects";
+import { ServiceId } from "../../../definitions/backend/ServiceId";
+import { markServiceAsRead } from "../../store/actions/services";
+import { isFirstVisibleServiceLoadCompletedSelector } from "../../store/reducers/entities/services/firstServicesLoading";
+
+/**
+ * A function to check if a service is loaded for the first time (at first startup or when the cache
+ * is cleaned). If true, the app shows the service list item without badge
+ * @param serviceId
+ */
+export function* handleServiceReadabilitySaga(serviceId: string): SagaIterator {
```

Comment:
```suggestion
export function* handleServiceReadabilitySaga(serviceId: ServiceId): SagaIterator {
```

Reply:
I suggest to not update from `string` to `ServiceId` within this pr and postpone it for a new pr: the string type is inherited from the attribute `sender_service_id` of both `CreatedMessageWithoutContent` and `CreatedMessageWithContent` ( used to load services related to messages). What about update both the message-related types and, then, update also che app code?

Reply:
@valentinacala 
We will change it in a new PR